web/ui/react-app/src/pages/targets/ScrapePoolList.tsx

```diff
@@ -1,45 +1,69 @@
 import React, { FC } from 'react';
-import { FilterData } from './Filter';
+import Filter, { Expanded, FilterData } from './Filter';
 import { useFetch } from '../../hooks/useFetch';
 import { groupTargets, Target } from './target';
 import ScrapePoolPanel from './ScrapePoolPanel';
 import { withStatusIndicator } from '../../components/withStatusIndicator';
 import { usePathPrefix } from '../../contexts/PathPrefixContext';
 import { API_PATH } from '../../constants/constants';
+import { useLocalStorage } from '../../hooks/useLocalStorage';
 
 interface ScrapePoolListProps {
-  filter: FilterData;
   activeTargets: Target[];
 }
 
-export const ScrapePoolContent: FC<ScrapePoolListProps> = ({ filter, activeTargets }) => {
+export const ScrapePoolContent: FC<ScrapePoolListProps> = ({ activeTargets }) => {
   const targetGroups = groupTargets(activeTargets);
+  const initialFilter: FilterData = {
+    showHealthy: true,
+    showUnhealthy: true,
+  };
+  const [filter, setFilter] = useLocalStorage('targets-page-filter', initialFilter);
+
+  const initialExpanded: Expanded = Object.keys(targetGroups).reduce(
+    (acc: { [scrapePool: string]: boolean }, scrapePool: string) => ({
+      ...acc,
+      [scrapePool]: true,
+      [scrapePool]: true,
```

Comment:
The doubling of this line is an accident, right?

Reply:
Yep, thanks for the catch. Fixed.